db/c_level_db.go

```diff
@@ -19,13 +19,15 @@ func init() {
 	registerDBCreator(CLevelDBBackendStr, dbCreator, false)
 }
 
+// CLevelDB provides access to the C implementation of LevelDB
```

Comment:
We should add a XXX comment that these methods panic on error from the underlying DB